internal/etw/eventopt.go

```diff
@@ -1,9 +1,20 @@
 package etw
 
+import (
+	"golang.org/x/sys/windows"
+)
+
+type eventOptions struct {
+	descriptor        *EventDescriptor

```

Comment:
Doesn't really matter but since you never allow this to be `nil` you dont need it to be a `ptr`.

Reply:
Going to treat this as Won't Fix for now, but will keep in mind in the future.